HISTORY.md

```diff
@@ -1,3 +1,9 @@
+4.0.7 / 2019-11-04
+
+  * deps: handlebars@4.5.1
+    - Bugfix: Contents of raw-blocks must be matched with non-eager regex-matching
+    - Bugfix: prevent zero length tokens in raw-blocks
```

Comment:
I think there are some missing things, looking at their release notes. For example, the change to `{{#each}}` isn't listed here, and I think that would make this a minor bump for us, right? Or would it not really count as a new feature available through this module?

Reply:
Added everything that looked important from the other releases in the release notes. 

A minor version bump makes sense to me but honestly I don't know much about the specific changes in handlebars.